.goreleaser.yml

```diff
@@ -8,8 +8,6 @@ builds:
     goarch:
       - amd64
       - arm64
-      - s390x
-      - ppc64le
```

Comment:
I think we should leave these. It doesn't cost us anything and people can use it in another image if they want to

Reply:
Per my comment on the other PR, does that cause confusion? If a user sees these in a list of binaries, they may think we support them. Then they go to install and it fails because the docker image doesn't work. Now they have to figure out why.

Reply:
I'm considering the case where someone just sees them, not even downloading them. They think we support that arch, so they go to install NGF on it and it fails. While it doesn't cost us anything to publish them, it could cost us questions.

Reply:
And it could actually cost us something if the pipeline fails to build them for whatever reason. While unlikely, if it happens, it's now something we have to fix for a platform we don't support.

Reply:
I'm going to move forward with this. Until we get feedback about these architecture, I think we just skip them altogether to avoid any questions or issues about them that *may* pop up, however unlikely.